docs/scripts/config.js

```diff
@@ -187,6 +187,11 @@ flag: \`${config.projectSlug} --config path/to/config.yaml\`.
 Config files can be formatted as JSON, YAML and TOML. Some configuration values support reloading without server restart.
 All configuration values can be set using environment variables, as documented below.
 
+This reference configuration documents all keys, also deprecated ones! 
```

Comment:
@vinckr this needs to be changed here: https://github.com/ory/docs/blob/master/scripts/config.js#L164

Reply:
https://github.com/ory/docs/pull/326

So this change in /hydra/docs is not necessary then right?

Reply:
yes